configs/13.0/deb/monolithic/runtime.postinst

```diff
@@ -24,7 +24,8 @@ if [[ "${1}" == configure ]]; then
 	[[ -f __AT_DEST__/sbin/ldconfig ]] && __AT_DEST__/sbin/ldconfig
 fi
 
-if [[ "__USE_SYSTEMD__" == "yes" && -n "$(command -v systemctl)" ]]; then
+if [[ "__USE_SYSTEMD__" == "yes" && -n "$(command -v systemctl)"\
+&& ! -e '/.dockerenv' && ! -e '/run/.containerenv' ]]; then
```

Comment:
This line should be indented, probably such that the `&&` lines up with `"__USE_SYSTEMD__".

Reply:
fixed